src/CSET/operators/plot.py

```diff
@@ -320,6 +320,76 @@ def _plot_and_save_line_series(
     plt.close(fig)
 
 
+def _plot_and_save_vertical_line_series(
+    cube: iris.cube.Cube,
+    coord: iris.coords.Coord,
+    filename: str,
+    title: str,
+    vmin: float,
+    vmax: float,
+    **kwargs,
+):
+    """Plot and save a 1D line series in vertical.
+
+    Parameters
+    ----------
+    cube: Cube
+        1 dimensional Cube of the data to plot on x-axis.
+    coord: Coord
+        Coordinate to plot on y-axis.
+    filename: str
+        Filename of the plot to write.
+    title: str
+        Plot title.
+    vmin: float
+        Minimum value for the x-axis.
+    vmax: float
+        Maximum value for the x-axis.
+    """
+    # plot the vertical pressure axis using log scale
+    fig = plt.figure(figsize=(8, 8), facecolor="w", edgecolor="k")
+    iplt.plot(cube, coord, "o-")
+    ax = plt.gca()
+    ax.invert_yaxis()
+    ax.set_yscale("log")
+
+    # Define y-ticks and labels for pressure log axis
+    y_tick_labels = [
+        "1000",
+        "850",
+        "700",
+        "500",
+        "300",
+        "200",
+        "100",
+        "50",
+        "30",
+        "20",
+        "10",
+    ]
+    y_ticks = [1000, 850, 700, 500, 300, 200, 100, 50, 30, 20, 10]
+
+    # Set y-axis limits and ticks
+    ax.set_ylim(1200, 100)
```

Comment:
```suggestion
    ax.set_ylim(1100, 100)
```
I don't think it needs to be as high as 1200 hPa, global record is 1083.8 hPa (from what I am aware) so suggest 1100 hPa as the upper limit.